include/certifier_framework.h

```diff
@@ -34,87 +34,60 @@ using std::string;
 
 namespace certifier {
   namespace framework {
+
+    // This will replace the old policy store
+    class store_entry {
+    public:
+      string tag_;
+      string type_;
+      string value_;
+
+      store_entry();
+      ~store_entry();
+
+      void print();
+    };
+
+    // Standard types are: string, binary-blob, der-encoded-cert, and protobuf serialized
+    //   key, keys, and signed-claim protobufs. However the store imposes no
+    //   restrictions on what serialization is.
     class policy_store {
     public:
-      enum {MAX_NUM_ENTRIES = 200};
-      bool policy_key_valid_;
+      enum {MAX_NUM_ENTRIES = 500};
+
+      int max_num_ents_;
+      int num_ents_;
+      store_entry** entry_;
 
+      bool policy_key_valid_;
       key_message policy_key_;
-      string encryption_algorithm_;
-
-      int max_num_ts_;
-      int num_ts_;
-      trusted_service_message** ts_;
-      int max_num_tsc_;
-      int num_tsc_;
-      tagged_signed_claim** tsc_;
-      int max_num_si_;
-      int num_si_;
-      storage_info_message** si_;
-      int max_num_tc_;
-      int num_tc_;
-      tagged_claim** tc_;
-      int max_num_tkm_;
-      int num_tkm_;
-      channel_key_message** tkm_;
-      int max_num_blobs_;
-      int num_blobs_;
-      tagged_blob_message** tagged_blob_;
 
     public:
-
+      policy_store(int max_ents);
       policy_store();
-      policy_store(const string enc_alg, int max_trusted_services, int max_trusted_signed_claims,
-          int max_storage_infos, int max_claims, int max_keys, int max_blobs);
       ~policy_store();
 
-      bool replace_policy_key(key_message& k);
-      const key_message* get_policy_key();
+      int get_num_entries();
```

Comment:
Consider returning `unsigned` ... for consistency with other interfaces that receive `unsigned ent` as the index-of-an-entry argument.